packages/libs/react-ui/src/components/Badge/Badge.css.ts

```diff
@@ -35,6 +35,35 @@ export const badge = recipe({
         borderColor: token('color.neutral.n1@alpha20'),
         color: token('color.neutral.n0@alpha70'),
       },
+      //kda/foundation/color/background/semantic/positive/inverse/default
+      info: {
+        backgroundColor: token(
+          'color.background.semantic.info.inverse.default',
+        ),
+        borderColor: token('color.neutral.n1@alpha20'),
+        color: token('color.neutral.n0@alpha70'),
```

Comment:
The color and borderColor are all the same right? Perhaps move those outside the variants?